ultimatejobweb/models.py

```diff
@@ -8,9 +7,20 @@ class user(models.Model):
     profession = models.CharField(max_length=128)
 
 
-class company(models.Model):
-    UrlCompany = models.URLField(max_length=200)
+class User_Search_History(models.Model):
```

Comment:
these are not the models we discussed yesterday

Reply:
It will be fixed today evening